docs/en/setup/service-agent/java-agent/Supported-list.md

```diff
@@ -70,6 +71,7 @@ metrics based on the tracing data.
   * [Apache CXF](https://github.com/apache/cxf) 3.x
   * [JSONRPC4J](https://github.com/briandilley/jsonrpc4j) 1.2.0 -> 1.6
   * [Nacos-Client](https://github.com/alibaba/nacos) 2.x (Optional²)
+  * [Netty](https://github.com/netty/netty) 4.1.x (Optional²)
```

Comment:
```suggestion
```

Reply:
> 

My negligence。According to https://github.com/apache/skywalking-java/pull/625#discussion_r1369424340，It is still in http server catalog better?